packages/shared/src/hooks/useShareComment.ts

```diff
@@ -27,7 +27,7 @@ export function useShareComment(origin: Origin): UseShareComment {
             ReferralCampaignKey.ShareComment,
           );
           await navigator.share({
-            text: `${post.title}\n${shortUrl}$}`,
+            text: `${post.title}\n${shortUrl}`,
```

Comment:
Should we even leave post title when they just copy the link not try to share it in squad or social media?

Reply:
Good point, we should keep it, but to do that, we should make the implementation similar to when sharing a post. Let me update it a little bit.

Reply:
With the applied changes, when you just hit copy on the native share, it won't add any link, then if you share it through something else, the title will be present.

Reply:
I tried to make it similar to `useSharePost` by utilizing the `title` property, but when you share, for example, on Twitter, the title is present on post share but on comment share, there is no preview. Not sure if we should make changes here now.

Reply:
I think we should stick to the original process until product says otherwise since we've had this flow since then.